tests/integration/utils.py

```diff
@@ -93,6 +94,42 @@ def wait_for_complete_request(self, response):
 
         return response
 
+    def fetch_all_requests(self, page=None, per_page=None, state=None, verbose=None):
+        """
+        Fetch a list of requests from the Cachito API.
+
+        :return: Object that contains response from the Cachito API
+        :rtype: list
+        """
+        page_tuple = ("page", page)
+        per_page_tuple = ("per_page", per_page)
+        state_tuple = ("state", state)
+        verbose_tuple = ("verbose", verbose)
+        url_values = urllib.parse.urlencode(
+            {
+                k: v
+                for k, v in [page_tuple, per_page_tuple, state_tuple, verbose_tuple]
+                if v is not None
+            }
+        )
+        result_url = f"{self._cachito_api_url}/requests?{url_values}"
```

Comment:
Just use the `json` parameter of `requests.get`. You could simplify this method a bit too:

```python
def fetch_all_requests(self, request_json=None):
  if request_json is None:
    request_json = {}

  resp = requests.get(result_url, json=request_json)
 ...
```

Then simply call it like:
```python
client.fetch_all_requests({'per_page': 1, 'page': 1, 'verbose': True})
```

Reply:
Thank you! Done.

Reply:
It seems like your last push didn't update it. Could you please address this again?

Reply:
Yes, sorry about that. Thank you for pointing.